### PR TITLE
Fix absolute source paths

### PR DESCRIPTION
**Pull request by Pospelove**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import { join } from "path";
+import { join, isAbsolute } from "path";
 import uploader from "./uploader";
 import purge from "./purge";
 import remove from "./remove";
 
 const run = async () => {
   try {
-    const source = join(
+    const source = isAbsolute(getInput("source")) ? getInput("source") : join(
```
Comment (ayeressian) [inline src/index.ts:9]: Can you assign the getInput("source") to a variable? To avoid two time calls.

**Comment by ayeressian** [comment]
Hi, @Pospelove can you also update the readme example to use version 2.0.2 and also the package json version to 2.0.2? Thanks!

**Comment by Pospelove** [comment]
Hi @ayeressian 
Done. I also removed the '=' sign in the readme example. the action didn't work for me without that.

**Review by ayeressian — APPROVED** [review]

**Comment by ayeressian** [comment]
> Hi @ayeressian Done. I also removed the '=' sign in the readme example. the action didn't work for me without that.

Yes, that was a typo. Thank you for fixing it.

**Comment by ayeressian** [comment]
@Pospelove FYI version 2.0.2 is released

**Comment by Pospelove** [comment]
@ayeressian thank you!

**Pull request merged**